src/main/java/com/getbase/models/Account.java

```diff
@@ -76,14 +76,14 @@ public void setTimezone(String timezone) {
   @Override
   public String toString() {
     return "Account{" +
-          "id=" + id + 
-          "createdAt=" + createdAt + 
-          "updatedAt=" + updatedAt + 
-          "currency='" + currency + '\'' + 
-          "name='" + name + '\'' + 
-          "phone='" + phone + '\'' + 
-          "timeFormat='" + timeFormat + '\'' + 
-          "timezone='" + timezone + '\'' + 
+          "id=" + id +
```

Comment:
maybe you could use https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/builder/ToStringBuilder.html ? or some similar builder?